core/src/test/java/de/danoeh/antennapod/core/storage/DbReaderTest.java

```diff
@@ -290,32 +291,68 @@ public void testGetNewItemIds() {
 
     @Test
     public void testGetPlaybackHistory() {
-        final int numItems = (DBReader.PLAYBACK_HISTORY_SIZE + 1) * 2;
-        final int playedItems = DBReader.PLAYBACK_HISTORY_SIZE + 1;
-        final int numReturnedItems = Math.min(playedItems, DBReader.PLAYBACK_HISTORY_SIZE);
-        final int numFeeds = 1;
+        // This for loop that manually calls setUp() and tearDown() is a hack
+        // because it seems that supporting a single parameterized test in a
+        // Robolectric test suite like this is difficult or impossible.
+        for (int offset: Arrays.asList(0, 10, 20)) {
```

Comment:
No idea why the CI check does not catch this, but please add a space here :)

```suggestion
        for (int offset : Arrays.asList(0, 10, 20)) {
```

Reply:
Done.